src/ImageSharp.Web/CaseHandlingUriBuilder.cs

```diff
@@ -181,6 +208,12 @@ private static void InitializeAbsoluteUriString(Span<char> buffer, (bool Lower,
                 pathBaseSpan = pathBaseSpan.Slice(0, pathBaseSpan.Length - 1);
             }
 
+            if (uriParts.Scheme.Length > 0)
+            {
+                index = CopyTextToBuffer(buffer, index, uriParts.Scheme.AsSpan());
```

Comment:
The scheme should always be case-insensitive:
```suggestion
                index = CopyTextToBufferLowerInvariant(buffer, index, uriParts.Scheme.AsSpan());
```

Reply:
Yeah, let's do that. 👍